### DISCUSSION TITLE
posts/SUSTechWifiLogin

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#9] [General]
# posts/SUSTechWifiLogin/

a student obsessed with the computing world

https://whexy.com/posts/SUSTechWifiLogin/

**Comment by drinktoomuchsax** [comment]
2021年的南科大二期宿舍网口不限速，少得折腾了

**Reply by whexy** [reply]
我也想住二期.jpg

**Reply by drinktoomuchsax** [reply]
但是最近校园网很辣鸡，各种限速、丢包。

**Reply by drinktoomuchsax** [reply]
最近手机疯狂重新登陆cas，记得看到过学长有写过自动登录的脚本，来找找思路

**Reply by drinktoomuchsax** [reply]
辣鸡校园网时代的泪水.png
